.changeset/fifty-cheetahs-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justifi/webcomponents": minor
```

Comment:
this should be a patch version, not minor

Reply:
Good call - fixed! Thank you